### PR TITLE
story-angular-bootstrap

### PR DESCRIPTION
**Pull request by burgerni10**

### CODE REVIEW

File: frontend/src/app/model/types.ts [hunk 1]
```diff
@@ -0,0 +1,215 @@
+import { DateTime } from 'luxon';
```
Comment (marouanehassanioptimistik) [inline frontend/src/app/model/types.ts:1]: this seems to contain many useless code :)

File: frontend/src/app/model/types.spec.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { AGGREGATION_FUNCTION_SETTINGS, ALL_AGGREGATION_FUNCTIONS } from './types';
+
+describe('types', () => {
```
Comment (marouanehassanioptimistik) [inline frontend/src/app/model/types.spec.ts:3]: is that needed ;)?

File: frontend/tsconfig.json [hunk 3]
```diff
@@ -10,6 +10,7 @@
     "noPropertyAccessFromIndexSignature": true,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
+    "strictNullChecks": false,
```
Comment (marouanehassanioptimistik) [inline frontend/tsconfig.json:13]: is this intended?

**Review by marouanehassanioptimistik — CHANGES_REQUESTED** [review]
i think we should group together all the directives required for form validation FormControlValidation, ValErrorDelayDirective 
@cexbrayat what is the best way to do that ?

**Comment by cexbrayat** [comment]
Maybe we can have an array declared somewhere like this:
```ts
export const formDirectives = [ReactiveFormsModule, FormControlValidation, ...]
```

and then use it in standalone components:
```
standalone: true,
imports: [formDirectives, ...]
```

File: frontend/src/app/model/types.spec.ts (hunk 2 above)
Comment (burgerni10) [reply frontend/src/app/model/types.spec.ts:3]: Indeed I can remove that for now. I added it in preparation of opcua scan groups implementation later.

**Pull request merged**